app/assets/stylesheets/components/_footer.sass

```diff
@@ -0,0 +1,11 @@
+$Footer-white: Theme-color(background, footer)
+$Footer-blue: Theme-color(color, footer)
+$Footer-height: $Theme-height-footer
+
+.Footer
+  background: $Footer-white
+
+.Footer-link
+  text-decoration: none
+  color: $Footer-blue
+  line-height: $Footer-height
```

Comment:
Since you're setting the line-height here for centering, maybe you should move the footer's height here as well.